src/main/scala/main.scala

```diff
@@ -127,6 +127,89 @@ object main extends App{
 
   println(iteration3("Fizz","Buzz",15))
   */
```

Comment:
please don't comment code out as it makes it super hard to read, if you don't want it to be executed just don't call the method